Intel_Extension_for_PyTorch_GettingStarted_and_AutoMixedPrecision_Sample/Intel_Extension_for_PyTorch_Getting_Started_and_AutoMixedPrecision.ipynb

```diff
@@ -8,35 +8,44 @@
    "outputs": [],
    "source": [
     "# =============================================================\n",
-    "# Copyright © 2020 Intel Corporation\n",
+    "# Copyright © 2024 Intel Corporation\n",
     "# \n",
     "# SPDX-License-Identifier: MIT\n",
     "# ============================================================="
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "b35a82c9-42b5-4d51-9c3f-ccffe6f8fdcd",
+   "metadata": {},
+   "source": [
+    "### NOTE: Before starting this notebook, please pick a jupyter kernel from dropdown from `pytorch-cpu` or `pytorch-gpu`. Choose the kernel depending on the hardware available to you. The `pytorch-gpu` kernel will only work on Intel® Flex/Max GPUs."
```

Comment:
I'd reduce the font size for this `Note`. I think `##` should be good here.

Reply:
Updated. To reduce the font size I added more `#`. I hope that's what you meant.